uint32 trans_seq
int32 trans_kind
int32 route_ps_num
RouteSP[] route_ps_info
